packages/forma-36-react-components/src/components/ControlledInput/README.mdx

```diff
@@ -0,0 +1,33 @@
+import { ControlledInput } from './ControlledInput';
+
+# ControlledInput
+
+The ControlledInput component is a styled form component that represents boolean state.
```

Comment:
I think we need to add some more context over here, but I recommend rising separate PR for this purpose. WDYT?

Reply:
Woops, this file wasn't supposed to go in this PR 😅

Reply:
Fixed. Good catch, thanks!